Source/WebCore/dom/TrustedType.cpp

```diff
@@ -170,6 +184,36 @@ ExceptionOr<String> trustedTypeCompliantString(TrustedType expectedType, ScriptE
     return stringValue;
 }
 
+String getTrustedTypeForAttribute(const String& tagName, const String& attributeParameter, const String& elementNamespace, const String& attributeNamespace)
+{
+    auto localName = tagName.convertToASCIILowercase();
+    auto attributeName = attributeParameter.convertToASCIILowercase();
+
+    if (attributeName.startsWith("on"_s))
```

Comment:
The spec for this has been updated to use the "event handler content attribute" concept. If you can easily match the attributeName to event handler names then great. If not feel free to leave this for now and we can address in a follow up.

Reply:
https://pr-preview.s3.amazonaws.com/lukewarlow/trusted-types/pull/457.html#get-trusted-type-data-for-attribute - latest preview of spec

Reply:
A follow-up bug is filed at https://bugs.webkit.org/show_bug.cgi?id=272675

Reply:
Code updated to be in line with the spec update. I will remove bug 272675 once this is landed.